quick-start/QuickStartTrackingApp/Views/MapView.swift

```diff
@@ -0,0 +1,175 @@
+            guard let pointAnnotation = annotation as? MLNPointAnnotation else {
+                return nil
+            }
+
+            let reuseIdentifier: String
+            var color: UIColor = .black
+            if pointAnnotation.accessibilityLabel == "Tracking" {
+                reuseIdentifier = "TrackingAnnotation"
+                color = UIColor(red: 0.00784313725, green: 0.50588235294, blue: 0.58039215686, alpha: 1)
+            } else if pointAnnotation.accessibilityLabel == "LocationChange" {
+                reuseIdentifier = "LocationChange"
+                color = .gray
+            } else {
+                reuseIdentifier = "DefaultAnnotationView"
+            }
+
+            var annotationView = mapView.dequeueReusableAnnotationView(withIdentifier: reuseIdentifier)
+
+            if annotationView == nil {
+                if reuseIdentifier != "DefaultAnnotationView" {
+                    annotationView = MLNAnnotationView(annotation: annotation, reuseIdentifier: reuseIdentifier)
+                    //If point annotation is an uploaded Tracking point the radius is 20 and color is blue, otherwise radius is 10 and color is gray
+                    let radius = pointAnnotation.accessibilityLabel == "Tracking" ? 20:10
+                    annotationView?.frame = CGRect(x: 0, y: 0, width: radius, height: radius)
+                    annotationView?.backgroundColor = color
+                    annotationView?.layer.cornerRadius = 10
+                    
+                    if pointAnnotation.accessibilityLabel == "Tracking" {
+                        annotationView?.layer.borderColor = UIColor.white.cgColor
+                        annotationView?.layer.borderWidth = 2.0
+                        annotationView?.layer.shadowColor = UIColor.black.cgColor
+                        annotationView?.layer.shadowOffset = CGSize(width: 0, height: 2)
+                        annotationView?.layer.shadowRadius = 3
+                        annotationView?.layer.shadowOpacity = 0.2
+                        annotationView?.clipsToBounds = false
+                    }
+                }
+                else {
+                    return nil
+                }
+            }
+
+            return annotationView
+        }
+        
+        func mapView(_ mapView: MLNMapView, didUpdate userLocation: MLNUserLocation?) {
+            if (userLocation?.location) != nil {
+                if trackingViewModel.trackingActive {
+                    let point = MLNPointAnnotation()
+                    point.coordinate = (userLocation?.location!.coordinate)!
+                    point.accessibilityLabel = "LocationChange"
+                    mapView.addAnnotation(point)
+                    Task {
+                        do {
+                            try await trackingViewModel.getTrackingPoints()
+                        }
+                        catch {
+                            print(error)
```

Comment:
Should we log something here instead of printing?

Reply:
Added alert and logger